calico-cloud/image-assurance/scanners/index.mdx

```diff
@@ -0,0 +1,11 @@
+---
+description: Scan images and workloads for vulnerabilities.
+hide_table_of_contents: true
+---
+
+# Honeypods
```

Comment:
Is this still a work in progress? I'm assuming it was a copy and paste error.

Reply:
fixed with https://github.com/tigera/docs/pull/676/commits/16d2c8cab26cc48b8ee6446e1aa77b017e8294ac